src/errors.ts

```diff
@@ -0,0 +1,27 @@
+interface WrappedError extends Error {
+  originalError?: unknown;
+  extensions?: Record<string, unknown>;
+}
+
+export interface OneSdkError extends Error {
+  properties?: Record<string, unknown>;
+  statusCode?: string;
+  metadata?: Record<string, unknown>;
+  response?: Record<string, unknown>;
+  kind?: string;
+}
+
+export function remapSdkError(originalError: OneSdkError): WrappedError {
+  const message = (originalError as Error)?.message || 'Unknown OneSDK Error';
```

Comment:
question: On this line I don't understand two things:

1. why there is type change to `Error`
2. why `?`